jenkins/nginx.sls

```diff
@@ -23,3 +25,18 @@ extend:
         - file: /etc/nginx/sites-available/jenkins.conf
       - require:
         - file: /etc/nginx/sites-enabled/jenkins.conf
+
+{% else %}
+
+Add nginx config for jenkins:
+  file.managed:
+    - template: jinja
+    - name: {{ jenkins.nginx_vhost_path }}/jenkins.conf
```

Comment:
Everything should assume a default, that way people can have a source of truth (defaults.yml) so they know what they can set